src/main/clojure/clara/rules/platform.cljc

```diff
@@ -75,4 +75,10 @@
            ~@body
            (finally
              ~@(for [[tl] binding-pairs]
-                 `(.remove ~tl)))))))
\ No newline at end of file
+                 `(.remove ~tl)))))))
+
+#?(:clj
```

Comment:
There isn't anything platform-specific here. `defmacro` is fine in cljs's compiler. So you shouldn't need to wrap it in this read conditional `:clj`.

Other places in this file use read conditionals due to using types that are platform-specific.